packages/compass-connections/src/components/non-genuine-connection-modal.tsx

```diff
@@ -23,6 +23,7 @@ export function showNonGenuineMongoDBWarningModal(
 ) {
   return showConfirmation({
     title: 'Non-Genuine MongoDB Detected',
+    hideCancelButton: true,
```

Comment:
I suggest hiding the cancel button on this as well, to keep things aligned and because canceling this modal won't stop connecting anyway.